lib/pgBackRest/Backup/Backup.pm

```diff
@@ -1096,12 +1096,12 @@ sub process
             {'strCipherPass' => $strCipherPassManifest}),
         $oStorageRepo->openWrite(
             "${strHistoryPath}/${strBackupLabel}.manifest." . COMPRESS_EXT,
-            {rhyFilter => [{strClass => STORAGE_FILTER_GZIP}],
+            {rhyFilter => [{strClass => STORAGE_FILTER_GZIP, rxyParam => [STORAGE_COMPRESS, false, 9]}],
```

Comment:
These files are relatively small but we keep them forever (there is no expiration for backup.history) so I decided maximum compression made sense.  I didn't bother with a constant since this code will be gone in a few months.